src/components/Notification/InactivityMessage.jsx

```diff
@@ -89,34 +94,60 @@ const InactivityMessage = () => {
         open={showPopup}
         aria-labelledby="inactivity-message-title"
         aria-describedby="inactivity-message-description"
+        PaperProps={{
+          sx: {
+            borderRadius: '25px'
+          }
+        }}
       >
-        <DialogTitle id="inactivity-message-title">Continue session?</DialogTitle>
-        <DialogContent id="inactivity-message-description">
-          <DialogContentText>
-            You have been inactive for a while now. Would you like to continue using PASS? You will
-            automatically be logged out in {Math.floor(secondsToLogout / 60)}:
-            {(secondsToLogout % 60).toLocaleString('en-US', {
-              minimumIntegerDigits: 2,
-              useGrouping: false
-            })}
-          </DialogContentText>
-        </DialogContent>
-        <DialogActions>
-          <LogoutButton onLogout={() => localStorage.clear()}>
-            <Button variant="outlined" color="error" endIcon={<LogoutIcon />} fullWidth>
-              Log Out
-            </Button>
-          </LogoutButton>
-          <Button
-            variant="contained"
-            color="primary"
-            endIcon={<CheckIcon />}
-            fullWidth
-            onClick={() => setShowPopup(false)}
-          >
-            Continue
-          </Button>
-        </DialogActions>
+        <Box
+          sx={{
+            display: 'flex',
+            flexDirection: 'column',
+            alignItems: 'center'
+          }}
+        >
+          <DialogTitle id="inactivity-message-title" sx={{ fontWeight: 'bold' }}>
+            CONTINUE SESSION?
```

Comment:
Make casing capitalized first letter rather than to all upper case?

Reply:
I believe there are other cases where `.toUpperCase()` is used for the dialog title. For those, think we could simply use the same formatting as the other modal titles, just capitalize the first letter.